### ISSUE TITLE
[Proposal] Противодействие мете против антагов с помощью механа.

### ISSUE THREAD

**Issue by Shudyn** [TauCetiStation/TauCetiClassic#5620]
Очевидно, что любой человек, у которого больше 10к минут и двух извилин уже давно понял, как определять режим по одному слову из радиоканала, и знает, чем отличаются режимы друг от друга. Люди, только услышав "Красные риги в бриге!", анонс о "хакере", или увидев "высосанный труп" уже точат баллоны и точно знают что делать, кого искать, чего остерегаться и т.д.

Что делать? 
Нужно обманывать и путать игроков, заставлять их думать "Ну очевидно сейчас малф", когда на самом деле блоб уже пожрал часть техов, а ИИ не виноват (или тритор).

Рандомные анонсы, которые могут/не могут говорить о режиме. Тот же "хакер", появляющийся только во время малфа, может быть, например, пойман отрядом с Икара, когда на деле готовятся воксы. Рандомные следы от режимов на станции, типа взломанного АПЦ или даже "высосанного трупа", который на деле является резиновой куклой с прошлой смены, оставленной ради рофлов. Те же самые картонки триторовские с изображениями алиумов или нюкеров, рандомно расставляемые в техах скрытым ивентом, но сгорающие при попытке экипажа с ними взаимодействовать. Можно даже добавить механ "показалось", чтобы с очень низкой вероятностью у рандомного космонавта на мгновение в поле зрение "входила" очевидная ролька и уходила обратно, или слышались анонсы или звуки, характерные для каких-то антагов. Дать раскрывшимся ролькам, типа генки с любой активной в данный момент способностью, какой-нибудь эмбиент и изредка рандомно его проигрывать. Вернется пусть и небольшая частичка страха неизвестного, как в первые тысячи минут игры, потому что даже ветераны не смогут знать чего ждать на этот раз. 

Да и эксперимент с триторами во время других режимов показывает, что как-никак это вводит разнообразие. Такого, как я помню, раньше не было. Поэтому стоит добавить вероятность смешивания других режимов. Только представьте, как нюкеры, вломившись в бриг, встречают там воксов (или другой отряд нюкеров, лол). Блоб и культ вместе. Или все и сразу с некоторым шансом. У антагов, все таки, в основном конфликтующие цели и им придется бороться за их выполнение между собой, строить шаткие союзы против общего врага и тому подобное.

Да-да, каждый раунд получит большой шанс скатиться в говно и КСку, но, если ты, борцун за хард-РоПэ и чай, не можешь РПшить во время говна, то может быть это проблема не игры? Люди не привыкли играть в такой обстановке, но они сами желают в ней побывать, как будто съесть запретный плод и хоть немного разнообразить пребывание в симуляторе реальной жизни в космосе. Вспоминается мой недавний рофл за главревера: "Приглашаются люди вступить в любую хуйню", и я сам же офигел обнаружив, что почти вся станция стала реверами, когда только-только прошел анонс на сокращение зарплат. "Это же старая лолфлеш рева,"- можешь сказать ты. Однако, людей никто не заставлял нажимать Yes в окошке предложения стать ревером.  А когда люди захотят поиграть в работку, они всего лишь проголосуют за эксту. Делов-то. Тем более нарушение написанных правил прямо пропорционально искушенности людей их нарушать. Дайте людям насытиться мясом, говном и кишкам до такой степени, что они устанут от них, и они сами начнут хорошо и по правилам играть в работку и хард-рп. 

ССка, как никак, все таки симулятор катастроф в глубоком космосе. Подобная фича увеличит количество возможных неповторимых ситуаций, делая каждый раунд немного особенным и запоминающимся. Именно абсурдность происходящего и делает эту игру интересной и часто забавной, а не повторяющиеся алгоритмы типа "построил сингу, ушел в крио/пить чай". И, как изначально поставленная цель, такой набор фич значительно снизит влияние меты на игру, так как никто не сможет точно знать, против какого антага их поставили. Ко всему быть готовым невозможно, иначе это просто манчкинизм. 

(Вероятно стоило эту стену текста на форум написать, а не сюда. Хотя я планировал только первые 4 абзаца. Может быть продублирую на форум, если будет смысл.)

Discuss.

**Comment by LudwigVonChesterfield** [comment]
Да, пожалуйста. Ни в коем случае не создавать однозначных "симптомов" режимов - аннонсы при малфе - это какая-то идиотия, бумажкуа при реве - это идиотя. Это даёт игрокам имеющим хоть сколько опыта с игрой однозначно и недвусмысленно понять какой режим.

Единственный способ добиться того чтобы игрок делал вид что он не знает о режиме - это не позволить ему о нём знать, никакие правила с этим так хорошо не справяться. Я полностью поддерживаю идею, и подписываюсь под каждым словом.

**Comment by Shudyn** [comment]
Нашел похожее обсуждение на форуме. В этом пропозале чуть больше раскрыл тему. https://forum.taucetistation.org/t/predlozhenie-po-tau-v-czelom/18486

**Comment by Motzord** [comment]
Уже было предложение делать аннонс о хакере при абсолютно любом взломанном ААПЦ, а ломать АПЦ умеют триторы, эми и нинжи, что уже не будет такой однозначной метой на режимы

**Comment by Motzord** [comment]
Насчет генокрадов, чтобы было меньше меты по трупам, достаточно
а) Вместо "бальшая арганическая игла" в отчете по вскрытии писать что-то "неизвестное" или выдавать "ошибку"
б) Не превращать космонавтика в куклу с хаском, дыркой в голове и нулем крови.

**Comment by LudwigVonChesterfield** [comment]
Какая разница что писать "игла", "ошибка", "неизвестное" - если это симптомы только у генокрада - всем и так будет понятно.

**Comment by Motzord** [comment]
Мета на культ и реву, смешная вещь.

Если культ палится своими очевидными "приглашениями в клуб художников/любую хуйню etc" или из-за того что рисуют кровью в коридорах, это не проблема так скажем режима.

Рева же палится из-за приказов ЦК, которые всем надоели, мне кажется данный механ "приказов" можно заменить бумажкой которая приходит вместе со сводкой таторов,  с глобальной директивой "Получите 100500 миллионов денег любым способом до конца смены"

**Comment by Motzord** [comment]
> Какая разница что писать "игла", "ошибка", "неизвестное" - если это симптомы только у генокрада - всем и так будет понятно.

не, ошибка выдается рандомно на трупах, часто ловил такой баг при вскрытии

**Comment by cool20141** [comment]
А зачем генкам уменьшать мету по трупам? Пусть трупы не палят просто

**Comment by Motzord** [comment]
> А зачем генкам уменьшать мету по трупам? Пусть трупы не палят просто

а как они не будут палить? Если труп останется в первозданном виде, то у генок будет другая проблема. "Еб вашу мать я видел труп чувака_нейм, его не клонировали, а он вон бегает!!"

**Comment by cool20141** [comment]
Выкинуть в космос, я так делаю всегда (Почти)

**Comment by DedYarick** [comment]
Хорошая идея, поддерживаю.

**Comment by KIBORG04** [comment]
> > Какая разница что писать "игла", "ошибка", "неизвестное" - если это симптомы только у генокрада - всем и так будет понятно.
> 
> не, ошибка выдается рандомно на трупах, часто ловил такой баг при вскрытии
> 
> (однако на гейплей он не влияет так что баг не страшный и ишью на гитхабе не нужно)

Если писалось "The ERROR", то это не баг. Там есть список рандомных хуевин, в который еррор и входит.

**Comment by Pivo24** [comment]
Микс режимов нужен

В качестве скрытия настоящих режимов ложными признаками других могу предложить следующее:

- Рандомные кровавые руны в заброшенных комнатах техтоннелей - и паранормалистам сразу есть что изучать, и ролькохантерам страшно
- Периодически вырубать свет в отдельных комнатках, как это делают тенелинги, и затем уже через, скажем, минуту, посылать анонс об электрическом сбое освещения
- Синий АПЦ триггерящий сообщение о хакере - очень хорошая идея
- ERROR вместо иглы будет хорошо, но чтобы не потерять ценность отчёта о вскрытии жертвы генокрада, почему бы не сделать так, что ERROR появляется только спустя 2-3 минуты лежания трупа.  Тогда появится смысл прятать трупы понадёжнее, чтобы никто не обнаружил их сразу же после убийства и не появился риск спалить своё присутствие на станции после вскрытия.
- Возможно, пара подрезанных проводков в начале раунда, мол с прошлой смены? И работа техтоннельным техникам, и прикрытие тем кто реально захочет резать проводку.

**Comment by cool20141** [comment]
Последнее точно нельзя, у нас такие долбоебы инженеры, что не могут различить соединенные провода от просто наложенных друг на друга, что уж говорить о поиске мест, где эти провода порезаны
